openshift/mlflow-experiment-job-template.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: Template
+
+labels:
+  application: ${APPLICATION_NAME}
+
+metadata:
+  name: mlflow-experiment-job
+
+parameters:
+- description: Application name
+  from: "experiment-run-[a-z0-9]{4}"
+  generate: expression
+  name: APPLICATION_NAME
+  required: true
+- name: MLFLOW_TRACKING_URI
+  value: http://$(MLFLOW_SERVER_SERVICE_SERVICE_HOST):$(MLFLOW_SERVER_SERVICE_SERVICE_PORT) # To use the internal host and port number to connect to the mlflow tracking server
```

Comment:
please add a description `URI to the MLFlow server. Only required if the MLFlow server is in another Namespace'

and the comment should go above that line